### PR TITLE
resolevd-ts: add resolved-ts related config doc

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6449
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1321,6 +1321,25 @@ Configuration items related to TiCDC.
 + Default value: `6`, which means 6 tasks can be concurrent executed at most.
 + Note: The value of `incremental-scan-concurrency` must be greater than or equal to that of `incremental-scan-threads`; otherwise, TiKV will report an error at startup.
 
+## resolved-ts
+
+Configuration items related to maintaining the Resolved TS to serve Stale Read requests.
+
+### `enable`
+
++ Enables whether to maintain the Resolved TS for all Regions.
```
Comment (TomShawn) [inline tikv-configuration-file.md:1330]:
```suggestion
+ Determines whether to maintain the Resolved TS for all Regions.
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1321,6 +1321,25 @@ Configuration items related to TiCDC.
...
++ Default value: `false`
```
Comment (TomShawn) [inline tikv-configuration-file.md:1331]:
```suggestion
+ Default value: `true`
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1321,6 +1321,25 @@ Configuration items related to TiCDC.
...
+
+### `advanced-ts-interval`
+
++ The time interval for regularly advancing the Resolved TS.
```
Comment (TomShawn) [inline tikv-configuration-file.md:1335]:
```suggestion
+ The time interval at which the Resolved TS is processed.
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1321,6 +1321,25 @@ Configuration items related to TiCDC.
...
++ Default value: `"1s"`
+
+### `scan-lock-pool-size`
+
++ The number of threads that TiKV scans for MVCC (multi-version concurrency control) lock data when initializing the Resolved TS.
```
Comment (TomShawn) [inline tikv-configuration-file.md:1340]: @NingLin-P PTAL whether this comment is correct or not. Thanks!
```suggestion
+ The number of threads that TiKV uses to scan the data locked by MVCC (multi-version concurrency control) when initializing the Resolved TS.
```

Comment (NingLin-P) [reply tikv-configuration-file.md:1340]: I think `MVCC (multi-version concurrency control) lock data` is more proper, TiKV only scan the lock record like `(key, lock info)`, and won't scan the `(key, value)` data

File: tikv-configuration-file.md (hunk 3 above)
Comment (NingLin-P) [reply tikv-configuration-file.md]: Perhaps we can reuse the description of `cdc.min-ts-interval`, they are used for the same goal.

**Review by NingLin-P — APPROVED** [review]
LGTM, thanks~

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**